### ISSUE TITLE
Do not support AndroidX

### ISSUE THREAD

**Issue by and-david** [dooboolab/flutter_inapp_purchase#62]
Build error with flutter_inapp_purchase package. I get the following error.

```
1: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
```
```
05: error: cannot find symbol
    public void onPurchasesUpdated(int responseCode, @Nullable List<Purchase> purchases) {
                                                      ^
  symbol: class Nullable
```

**Comment by hyochan** [comment]
I haven't had any idea about `AndroidX` yet. How should I handle this?

**Comment by RomanBase** [comment]
Just remove @Nullable and other annotations and it should be OK.

**Comment by ened** [comment]
@RomanBase no, those should be kept.
@hyochan See how other plugins have done it in the past, like https://github.com/JackAppDev/flutter_iap/commit/85eda38e42fad2737970c4448a3429b6c7eaac73

**Comment by hyochan** [comment]
@ened That's great. There is already a solution for it.

Could someone try out for `PR` to maintain this?
I think I'll be available in 2 weeks but I can be a quick reviewer.

**Comment by JeffScaturro** [comment]
Until the pull request above is merged, you can consume Android X support via a dependency override in your `pubspec.yaml`:

```yaml
dependency_overrides:
  flutter_inapp_purchase:
    git:
      url: git@github.com:AnyKindaApp/flutter_inapp_purchase.git
```

**Comment by stemuk** [comment]
@JeffScaturro Thanks for the quick fix! If I try to use the dependency overwrite however I get this:
```
Git error. Command: git clone --mirror git@github.com:AnyKindaApp/flutter_inapp_purchase.git /home/stefan/.pub-cache/git/cache/flutter_inapp_purchase-9d5175f69e943d9938ffa7d63cd0cc51a46287cd
Cloning into bare repository '/home/stefan/.pub-cache/git/cache/flutter_inapp_purchase-9d5175f69e943d9938ffa7d63cd0cc51a46287cd'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Or am I doing something wrong on my end?

**Comment by JeffScaturro** [comment]
@stemuk I pasted the ssh version so you'd have to be authenticated with git already (something like `ssh-add -K ~/.ssh/id_rsa`). I updated the original comment above to include the https version which is probably more preferable. Sorry about that.

**Comment by hyochan** [comment]
Merged #65 Thank you @JeffScaturro !
